### ISSUE TITLE
Readme instructions for gstreamer codecs

### ISSUE THREAD

**Issue by cxkoda** [pdfpc/pdfpc#380]
After installing the main dependencies and compiling pdfpc on Ubuntu 18.04 I got this error displaying a pdf with linked movie.
`Error creating video sink: Could not make element sink0 of type gtksink.`
This was solved by installing the package `gstreamer1.0-gtk3`.
Could you include this package in the main dependencies?

Especially since the optional command in the readme
`sudo apt-get install gstreamer1.0-*`
does not work out of the box. It returns on my setup
```
The following packages have unmet dependencies:  
gstreamer1.0-python3-dbg-plugin-loader : Conflicts: gstreamer1.0-python3-plugin-loader but 1.14.1-1~ubuntu18.04.1 is to be installed  
gstreamer1.0-python3-plugin-loader : Conflicts: gstreamer1.0-python3-dbg-plugin-loader but 1.14.1-1~ubuntu18.04.1 is to be installed  
E: Unable to correct problems, you have held broken packages.
```

pdfpc version: 5f4e9a64a9a64274766526d0d0babacacfa89808

**Comment by AndreasBilke** [comment]
We can enforce compile dependencies via cmake. But the Ubuntu issue is a run time dependency problem. It's the business of the Ubuntu package maintainers. Moreover: The current Debian package for testing (which Ubuntu just copies once in a while) includes the missing dependency.

You are free to complain at the Ubuntu bug tracker that they do *something*.

I also doubt that some README instructions at our side would help, since usually this is read seldom.

**Comment by fnevgeny** [comment]
I think the report was about compiling from sources, not using the Ubuntu package.

**Comment by AndreasBilke** [comment]
@cxkoda Can you propose a better section at the README to build it on Ubuntu?

**Comment by AndreasBilke** [comment]
See @fnevgeny proposal at #381

**Comment by cxkoda** [comment]
> 
> 
> 
> Sorry, I think now I understand you. You used our README to build it from source and then you got the problems with missing dependencies?

Yes, exactly.
> 
> 
> Fixed in #381. @cxkoda sorry for my misunderstanding.

No problem. Thanks for the fast update.
